processors/config/anaKalSimpTuple_cfg.py

```diff
@@ -105,8 +105,9 @@
     print ("ERROR::Need to specify which tracks KF or GBL")
     exit(1)
 
-if (options.nevents > 0):
-    p.max_events = options.nevents
+p.skip_events = options.skip_events
+if(options.nevents>-1):
```

Comment:
You are not setting p.max_events when options.nevents = -1

Reply:
I think if I dont set it, it will run over everything automatically